lib/squib/dsl/avatar.rb

```diff
@@ -0,0 +1,92 @@
+require 'mechanize'
+require_relative '../errors_warnings/warn_unexpected_params'
+require_relative '../args/card_range'
+require_relative '../args/paint'
+require_relative '../args/scale_box'
+require_relative '../args/transform'
+require_relative '../args/svg_special'
+
+module Squib
+  class Deck
+    def avatar(opts = {})
+      DSL::Avatar.new(self, __callee__).run(opts)
+    end
+  end
+
+  module DSL
+    # Add an avatar for placeholder art in your games
+    # using https://avatars.dicebear.com/. The image will
+    # be downloaded to your configured image directory if
+    # it doesn't already exist.
+    #
+    # Library can be male, female, human, identicon, initials,
+    # bottts, avataaars, jdenticon, gridy or micah.
+    #
+    # Seed can be any random string
+    #
+    # Example:
+    #   avatar library: 'micah', seed: '1234'
+    class Avatar
+      include WarnUnexpectedParams
+      attr_reader :dsl_method, :deck
+
+      def initialize(deck, dsl_method)
+        @deck = deck
+        @dsl_method = dsl_method
+      end
+
+      def self.accepted_params
+        %i[
+          library seed
+          x y width height
+          blend mask
+          crop_x crop_y crop_width crop_height
+          crop_corner_radius crop_corner_x_radius crop_corner_y_radius
+          flip_horizontal flip_vertical angle
+          range layout
+        ]
+      end
+
+      def run(opts)
+        warn_if_unexpected opts
+        Dir.chdir(deck.img_dir) do
+          defaults = { library: 'avataaars' }
```

Comment:
```suggestion
          defaults = { library: 'avataaars', seed: 'squib-default-seed' }
```

I like to have *everything* have a default so things work even if you forget something. This seemed most sensible